main.go

```diff
@@ -63,13 +63,15 @@ func installedNdkVersion(ndkHome string) string {
 }
 
 func currentNdkHome() string {
-	if v := os.Getenv(androidNDKHome); v != "" {
+	if v := os.Getenv(androidNdkHome); v != "" {
 		return v
 	}
 	if v := os.Getenv("ANDROID_HOME"); v != "" {
+		// $ANDROID_HOME is deprecated
 		return filepath.Join(v, "ndk-bundle")
 	}
 	if v := os.Getenv("ANDROID_SDK_ROOT"); v != "" {
+		// $ANDROID_SDK_ROOT is preferred over $ANDROID_HOME
```

Comment:
If it is preferred over ANDROID_HOME it should be checked before.
If understand this fragment correctly now, if ANDROID_HOME is defined then function will return without checking this.

Reply:
I want this to behave the same way as our go-android package (which is used in different steps): https://github.com/bitrise-io/go-android/blob/master/sdk/sdk.go#L58

`$ANDROID_HOME` is currently defined on our stacks by default, so this code shouldn't cause any problem now. We want to start using `ANDROID_SDK_ROOT` one day (both in this step and on the stack), but that will require a bit more investigation and careful planning to not break anything, including older Android projects.